foundation/foundation-upgrade/src/main/resources/migrations.json

```diff
@@ -73,7 +73,8 @@
         "implementation": "com.boozallen.aissemble.upgrade.migration.v1_10_0.DeltaSparkYamlMigration",
         "fileSets": [
           {
-            "includes": ["**/*.yaml"]
+            "includes": ["**/*.yaml"],
+            "excludes": ["**/target/**", "**/templates/**"]
```

Comment:
I: Not strictly necessary but this cuts down significantly on the number of output lines talking about un-parseable helm templates.